### ISSUE TITLE
Row actions popover not accessible

### ISSUE THREAD

**Issue by sheribyrne** [vmware/clarity#3280]
http://clarity-ks.surge.sh/datagrids
Multi-select with batch actions and single row actions

The vertical ellipses behavior is not fully accessible.  Violations are:

WCAG 1.4.13 pop-up is obscuring calling control
WCAG 2.1.1 list is not keyboard traversable
WCAG 2.4.7 List requires keyboard focus indicator that meets 3.0:1 ratio as required in 1.4.11 (non text contrast minimums)

**Comment by gnomeontherun** [comment]
Probably a duplicate of #3371, will verify as part of that work.

**Comment by gnomeontherun** [comment]
This should have been addressed as part of various accessibility fixes for the Datagrid, closing as resolved.
